main/main.py

```diff
@@ -78,6 +82,7 @@ def main():
             publish_all_artifacts_to_binaries(artifactory, binaries, rr, buildinfo)
             set_output("publish_to_binaries", f"{repo}:{version} publish_to_binaries DONE")
 
+        burgr = Burgr(burgrx_url, burgrx_user, burgrx_password, rr)
         burgr.notify(buildinfo, 'passed')
```

Comment:
@tomverin: I want to remove it but I was wondering if this would not break a feature in BURGR (perhaps it is already broken)

Reply:
I double checked and we can remove this code